include/deal.II/base/numbers.h

```diff
@@ -681,8 +681,8 @@ namespace internal
   template <typename T>
   struct NumberType
   {
-    static constexpr DEAL_II_CUDA_HOST_DEV const T &
-                                                 value(const T &t)
+    static DEAL_II_ALWAYS_INLINE constexpr DEAL_II_CUDA_HOST_DEV const T &
+                                                                       value(const T &t)
```

Comment:
Here is an example of weird indentation of `clang-format`. While I understand what it tries to do, it just gets nasty (and beyond the 80 char limit several times).

Reply:
can you force a line break by inserting ``/* bla */`` before ``const`` for example?

Reply:
I tried a few things and while I could improve the situation sometimes, this was not consistently the case. I therefore suggest we leave the discussion about improving indentation to a more systematic approach - I have opened #8437 for that.